packages/manager/modules/ng-layout-helpers/package.json

```
@@ -10,12 +10,27 @@
   },
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
-  "main": "index.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/manager-ng-layout-helpers.js",
```

Isn't supposed to be:
```suggestion
  "browser": "./dist/umd/ng-layout-helpers.js",
```

---

So the package name should be updated to `@ovh-ux/ng-layout-helpers` ? Seems to be manager specific no ?

---

@cbourgois package name is different from the name generated while building the dist: it takes the name of the folder and not the name of the package